rbi/annotations/railties.rbi

```diff
@@ -26,27 +26,15 @@ module Rails
 end
 
 class Rails::Application < ::Rails::Engine
-  sig { returns(Rails::Application::Configuration) }
+  sig { returns(T.untyped) }
   def config; end
 end
 
-module Rails::Command::Behavior
```

Comment:
Are these being removed because Tapioca can generate them?

Reply:
Yes. At least this is what the CI on the previous commit told me.